djangoproject/templates/members/individualmember_list.html

```diff
@@ -5,12 +5,37 @@
   <h1>{% trans "Individual members" %}</h1>
   <p>
     {% blocktrans %}
-      The following are Individual Members of the Django Software Foundation. Individual Members
-      are appointed by the DSF in recognition of their service to the Django community. If you
-      know someone who you think should be considered for Individual Membership, please
+      Individual Members are appointed by the DSF in recognition of their service to the Django community.
+
+      Service to the Django community takes many forms. Here are some (non-exhaustive) examples of the categories of work that might qualify as service:
```

Comment:
No need for parentheses, be bold!
```suggestion
      Service to the Django community takes many forms. Here are some non-exhaustive examples of the categories of work that might qualify as service:
```